neuralpredictors/layers/readouts/base.py

```diff
@@ -19,110 +12,66 @@ class ConfigurationError(Exception):
 
 
 class Readout(nn.Module):
```

Comment:
It seems all the readouts have the `regularizer()` method. And I think the Multi-readout expects it. In that case, I suggest we add it to the base class where it returns `0.0`.

Reply:
Good point. I would add a NotImplementedError version of the regularizer method though, instead of returning 0.0. What do you think? People can still easily just make it return 0 but they should be aware of it.

Reply:
@eywalker @KonstantinWilleke what do you guys think?

Reply:
I agree better make it fail than silently potentially not work.